src/page/detail/detail.tsx

```diff
@@ -9,12 +9,22 @@ import {
 } from '@fortawesome/free-solid-svg-icons';
 import {styles} from './detailCss';
 import Logo from '../../resource/images/logo.svg';
+import {useNavigation, useRoute} from '@react-navigation/native';
 
-const Detail: React.FC = (Props: Props) => {
+const Detail: React.FC = () => {
+  const route = useRoute();
+  const navigation = useNavigation();
+  const param = route.params && route.params.detail;
+
+  function goback(): void {
```

Comment:
You don't need set `void` manually.
Also please put all function in `useCallback` function
```
const goBack = useCallback(() => {
 navigation.goBack();
},[ navigation ])
```